lib/src/ast.dart

```diff
@@ -81,26 +81,12 @@ bool hasConstantError(LinterContext context, Expression node) {
 }
 
 /// Returns `true` if this [element] has a `@literal` annotation.
-bool hasLiteralAnnotation(Element element) {
-  var metadata = element.metadata;
-  for (var i = 0; i < metadata.length; i++) {
-    if (metadata[i].isLiteral) {
-      return true;
-    }
-  }
-  return false;
-}
+@Deprecated('prefer: element.hasLiteral')
```

Comment:
It looks like you already removed references from this package. Is there any reason not to remove these two functions?

Reply:
It *might* be used internally?